components/admin/requests/processing/TasksCard.tsx

```diff
@@ -70,28 +70,28 @@ export default function ProcessingTasksCard({ applicationId }: ProcessingTasksCa
     refetch();
   };
 
-  const [assignInvoiceNumber] = useMutation<
-    AssignInvoiceNumberResponse,
-    AssignInvoiceNumberRequest
-  >(ASSIGN_INVOICE_NUMBER_MUTATION);
-  const handleAssignInvoiceNumber = async (invoiceNumber: number) => {
-    await assignInvoiceNumber({ variables: { input: { applicationId, invoiceNumber } } });
-    refetch();
-  };
+  // const [assignInvoiceNumber] = useMutation<
+  //   AssignInvoiceNumberResponse,
+  //   AssignInvoiceNumberRequest
+  // >(ASSIGN_INVOICE_NUMBER_MUTATION);
+  // const handleAssignInvoiceNumber = async (invoiceNumber: number) => {
+  //   await assignInvoiceNumber({ variables: { input: { applicationId, invoiceNumber } } });
+  //   refetch();
+  // };
 
-  const [uploadDocuments] =
-    useMutation<UploadDocumentsResponse, UploadDocumentsRequest>(UPLOAD_DOCUMENTS_MUTATION);
-  const handleUploadDocuments = async (documentsUrl: string) => {
-    await uploadDocuments({ variables: { input: { applicationId, documentsUrl } } });
-    refetch();
-  };
+  // const [uploadDocuments] =
+  //   useMutation<UploadDocumentsResponse, UploadDocumentsRequest>(UPLOAD_DOCUMENTS_MUTATION);
+  // const handleUploadDocuments = async (documentsUrl: string) => {
+  //   await uploadDocuments({ variables: { input: { applicationId, documentsUrl } } });
+  //   refetch();
+  // };
```

Comment:
Just wondering why these are commented?

Reply:
Initially, the assignInvoiceNumber mutation was commented out because the assign invoice number step was replaced by the generate invoice step so I wasn't sure if the mutation was needed but I didn't want to delete it yet in case we still needed it. UploadDocuments was commented because I assumed we were going to use our recently done AWS doc upload stuff in replacement but was also not sure if I should delete it yet.